src/drivers/imu/adis16477/ADIS16477.hpp

```diff
@@ -148,12 +92,12 @@ class ADIS16477 : public device::SPI, public px4::ScheduledWorkItem
 	/**
 	 * Start automatic measurement.
 	 */
-	void		start();
+	void			start();
```

Comment:
Hi @dagar, I feel more and more convinced every month that we need to switch to spaces, not tabs, because of how many of these changes get into PRs every month.  Also, I think a single space between the type and the name is sufficient.... particularly if someone has taken the time to doxy document it.

Reply:
> I feel more and more convinced every month that we need to switch to spaces, not tabs,

🚫 😃 

> I think a single space between the type and the name is sufficient....

I agree, I also follow this convention.